Doc/library/stdtypes.rst

```diff
@@ -3568,14 +3573,14 @@ produce new objects.
       [b'1', b'2', b'3']
 
 
-.. method:: bytes.strip([chars])
-            bytearray.strip([chars])
+.. method:: bytes.strip(bytes=None, /)
+            bytearray.strip(bytes=None, /)
 
    Return a copy of the sequence with specified leading and trailing bytes
-   removed. The *chars* argument is a binary sequence specifying the set of
+   removed. The *bytes* argument is a binary sequence specifying the set of
    byte values to be removed - the name refers to the fact this method is
-   usually used with ASCII characters.  If omitted or ``None``, the *chars*
-   argument defaults to removing ASCII whitespace. The *chars* argument is
+   usually used with ASCII characters.  If omitted or ``None``, the *bytes*
```

Comment:
this sentence no longer makes sense after changing the argument name

Reply:
On other hand, `bytes` is already used in the signature for long time. So this is in the scope of this PR.